test/e2e/e2e_test.go

```diff
@@ -111,4 +123,33 @@ var _ = Describe("e2e control plane", func() {
 
 		framework.ExpectNoError(<-errChan)
 	})
+
+	ginkgo.It("should provide Internet connection continuously when master is killed", func() {
+		ginkgo.By("Running container which tries to connect to 8.8.8.8 in a loop")
+
+		podChan, errChan := make(chan *v1.Pod), make(chan error)
+		go checkContinuousConnectivity(f, "", "connectivity-test-continuous", "8.8.8.8", 53, 30, podChan, errChan)
+
+		testPod := <-podChan
+		framework.Logf("Test pod running on %q, waiting a few seconds", testPod.Spec.NodeName)
+
+		time.Sleep(10 * time.Second)
```

Comment:
What is this for? There are helper routines in the kubernetes e2e framework for, eg, waiting until a pod reports a Ready status, etc. That's faster when the cluster is fast and reliable when the cluster is slow.

Reply:
I just removed the wait, since the checkContinuousConnectivity function now waits until the pod exit the pending state before allowing the test to continue.